determined-ai/determined
# fix: `GetModelVersion` lookup by version number.

## Description

I registered a model in model registry, tried to run "Use in notebook" code snippet and discovered that `model.get_version` method in python SDK returns 404. 
Turns out, the test and SQL were using table id instead of the version number for lookup, and the integration test was testing it incorrectly as well.

## Test Plan

1. Register a model in model registry.
2. Go to the registered version.
3. Click "use in notebook", copy the snippet.
4. Run a notebook and paste the snippet into it.
5. It should work.

## Commentary (optional)

## Checklist
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
